templates/rhpam73-managed.yaml

```diff
@@ -1179,30 +1185,30 @@ objects:
         volumes:
         - name: "${APPLICATION_NAME}-postgresql-pvol"
           persistentVolumeClaim:
-            claimName: "${APPLICATION_NAME}-postgresql-claim-1"
-## PostgreSQL deployment config 1 END
-## PostgreSQL persistent volume claim 1 BEGIN
+            claimName: "${APPLICATION_NAME}-postgresql-claim"
+## PostgreSQL deployment config END
+## PostgreSQL persistent volume claim BEGIN
 - apiVersion: v1
   kind: PersistentVolumeClaim
   metadata:
-    name: "${APPLICATION_NAME}-postgresql-claim-1"
+    name: "${APPLICATION_NAME}-postgresql-claim"
     labels:
       application: "${APPLICATION_NAME}"
-      service: "${APPLICATION_NAME}-postgresql-1"
+      service: "${APPLICATION_NAME}-postgresql"
   spec:
     accessModes:
     - ReadWriteOnce
     resources:
       requests:
         storage: "${DB_VOLUME_CAPACITY}"
-## PostgreSQL persistent volume claim 1 END
+## PostgreSQL persistent volume claim END
 - apiVersion: v1
   kind: PersistentVolumeClaim
   metadata:
     name: "${APPLICATION_NAME}-rhpamcentr-claim"
     labels:
       application: "${APPLICATION_NAME}"
-      service: "${APPLICATION_NAME}-rhpamcentrmon"
+      service: "${APPLICATION_NAME}-rhpamcentr"
```

Comment:
@errantepiphany This label should be same as before, because it's used for other object related to rhpmacentrmon (pods, dc, services, routes, ...).

Reply:
@jakubschwan Ah, I see that now, thanks. Nice catch.

Reply:
@jakubschwan Addressing on this PR: https://github.com/jboss-container-images/rhpam-7-openshift-image/pull/257/files